<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>splashScreen</class>
 <widget class="QWidget" name="splashScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>381</width>
     <height>191</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/iXServiceProvaider.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="systemLoadInfo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>431</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
color: rgb(0, 170, 255);
font: 75 9pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>正在加载应用程序资源...</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
